README.md

```diff
@@ -801,6 +801,32 @@ ctx.run(query[Book])
 // SELECT id, notes, pages, history FROM Book
 ```
 
+### User-Defined Types
+
+The cassandra context provides encoding of UDT (user-defined types).
+```scala
+import io.getquill.context.cassandra.Udt
+
+case class Name(firstName: String, lastName: String) extends Udt
+```
+
+It's possible to override UDT name or columns using `udtMeta`:
+```scala
+implicit val nameMeta = udtMeta[Name]("my_name", _.firstName -> "first")
+```
+
+To translate UDT name/columns from case classes, the cassandra context first tries to use definitions from `udtMeta`
+and only if those could not be found, then translates them using `NamingStrategy`.
+
+It's also possible to specify UDT keyspace using `udtMeta`:
+
+```scala
+implicit val nameMeta = udtMeta[Name]("keyspace2.my_name", _.lastName -> "last_name")
+```
+
+When keyspace is not explicitly set, context looks for UDT within all keyspaces.
```

Comment:
According to cql documentation: "The scope of a user-defined type is the keyspace in which you define it.". So our UDT must be defined in the same keyspace our tables belong to. Also all CassandraContext implementations (except mirror) require working keyspace for initialization. It seems no need to scan all keyspaces.

Reply:
@optician it's possible to open session which not bound to any keyspace, so you can access several of them. It's simple as `cluster.connect()` or `cluster.connect(null)` (it's equals). You can also do it with quill by passing null for session param in constructor

Reply:
@optician But it's also make sense to not look for UDT within other keyspaces if context set with concrete keyspace. Truly saying, I'm not 100% sure to which direction should we move.

What if:
if keyspace is set then match UDT only from this one.
otherwise, if keyspace is null - look for UDT within all cluster?